app/assets/stylesheets/_size.scss

```diff
@@ -0,0 +1,2 @@
+$nav_height: 50px;
+$icon_size: 36px;
```

Comment:
Name of variable `icon_size` should be written in all lowercase letters with hyphens instead of underscores